ClientRuntimes/CSharp/Microsoft.Rest.ClientRuntime/project.json

```diff
@@ -24,6 +24,9 @@
         "System.Net.Http": "",
         "System.Net.Http.WebRequest": "",
         "System.Xml": ""
+      },
+      "dependencies": {
+        "Newtonsoft.Json": "6.0.8"
```

Comment:
JSON.NET does fix bugs from time to time. I reported an issue that was fixed in 7.0.1: https://github.com/JamesNK/Newtonsoft.Json/issues/672  It doesn't address a common scenario, but it has affected some of our customers in the past.

More generally, I don't think it's good policy to freeze dependency versions for all time. The question shouldn't really be *if*, but *when* should it be updated.